LocalSettings.php

```diff
@@ -874,21 +874,22 @@
+				'Categorized as private' => 'The purpose and scope of your wiki is clear enough. Please ensure your wiki complies with all aspects of the Content Policy and Code of Conduct at all times or it may be closed. Please also note that I have categorized your wiki as "Private". Thank you.',
 			],
 			'Decline reasons' => [
-				'Needs more details' => 'Can you give us a few more details on the purpose for, scope of, and topic of your wiki, and briefly describe some of your wiki\'s content in approximately 2-3 sentences? Additionally can you elaborate on your wiki\'s scope and topical focus a bit further? A few sentences describing the scope of your wiki and the sort of content it will contain should be helpful. Please go back into your original request and add to, but do not replace, your existing description. Thank you.',
-				'Invalid or unclear subdomain' => 'The scope and purpose of the wiki seem clear enough. However, your requested subdomain is either invalid, is too generic, conveys a Miraheze affiliation, or suggests the wiki is an English language or multilingual wiki when it is not. Please change it to something that better reflects your wiki\'s purpose and scope. Thank you.',
-				'Invalid sitename/subdomain (obscene wording)' => 'The scope and purpose of the wiki seem clear enough. However, the requested wiki name or subdomain is in violation of our Content Policy which prohibits obscene wording in wiki names and subdomains. Please change it to something that is better. Thank you.',
+				'Needs more details' => 'Can you give us more details on the purpose for, scope of, and topic of your wiki, ideally in at least 2-3 sentences? Please go back into your original request and add to, but do not replace, your existing description. Thank you.',
+				'Invalid or unclear subdomain' => 'The scope and purpose of your wiki seem clear enough; however, the requested subdomain is either invalid, too generic, conveys a Miraheze affiliation, or suggests that the wiki is an English language or multilingual wiki when it is not. Please change it to something that better reflects your wiki\'s purpose and scope. Thank you.',
+				'Invalid sitename/subdomain (obsence wording)' => 'The scope and purpose of your wiki seem clear enough; however, the requested wiki name or subdomain is in violation of our Content Policy, which prohibits obsence wording in wiki names and subdomains. Please change it to something that is appropriate. Thank you.',
```

Comment:
"obsence" is not a word (I had to do #5326 to fix this previously), and you are once again creating a big sentence. The "better" -> "appropriate" change makes sense to me though.

Reply:
Fixed; not sure how "obsence" got in there.